authors/apps/profiles/views.py

```diff
@@ -61,3 +61,68 @@ class UserProfileListView(generics.ListAPIView):
     @classmethod
     def get_queryset(self):
         return Profile.objects.all()
+
+
+class FollowersAPIView(generics.RetrieveAPIView):
+    """A class for actors who request.user is following."""
+    permission_classes = (IsAuthenticated,)
+    serializer_class = FollowerSerializer
+    lookup_field = 'username'
+
+    @classmethod
+    def get(cls, request, username):
+        # returns a list of actors who follow request.user
+        following = Follower.objects.values_list(
+            'follow', flat=True).filter(
+            author=username)
+        response = {'message': following, 'status_code': status.HTTP_200_OK}
+        return Response(
+            response,
+            content_type='text/json',
+            status=status.HTTP_200_OK)
+
+
+class FollowCreateAPIView(generics.CreateAPIView):
+    """A class for following other authors."""
+    permission_classes = (IsAuthenticated,)
+    lookup_field = 'username'
+    serializer_class = FollowerSerializer
+
+    @classmethod
+    def post(cls, request, username):
+        # Follow the author (where an author username=username).
+        data = {'author': request.user.username, 'follow': username}
```

Comment:
@ManuelDominic , you don't need to add the author to the serialiser, you can only add it when you are saving.
Putting it in the serialiser means a user has to add it to the request body.